### PR TITLE
Remove import from bootstrap esm

### PR DESCRIPTION
**Pull request by trondal**
## Purpose

* Removed wrong import from Bootstrap

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by ghost** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/Azure-Samples/ms-identity-javascript-react-spa?pullRequest=31) <br/>All CLA requirements met.

File: src/components/SignInButton.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { useMsal } from "@azure/msal-react";
 import { loginRequest } from "../authConfig";
 import DropdownButton from "react-bootstrap/DropdownButton";
-import Dropdown from "react-bootstrap/esm/Dropdown";
+import Dropdown from "react-bootstrat/Dropdown";
```
Comment (derisen) [inline src/components/SignInButton.jsx]:
```suggestion
import Dropdown from "react-bootstrap/Dropdown";
```

**Review by derisen — CHANGES_REQUESTED** [review]
@trondal thanks for catching this! One comment to address

**Review by derisen — APPROVED** [review]
Thanks @trondal , LGTM. It seems you also need to sign the CLA -see the bot [comment](https://github.com/Azure-Samples/ms-identity-javascript-react-spa/pull/31#issuecomment-1283922947). You should be able to omit the company field if that's not relevant.

**Comment by trondal** [comment]
@microsoft-github-policy-service agree [company="{default}"]

**Comment by trondal** [comment]
@microsoft-github-policy-service agree

**Pull request merged**